test/property.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+var test = require('tape')
+var fc = require('fast-check')
+var _ = require('lodash')
+var is = require('..')
+
+test('unist-util-is properties', function (t) {
+  t.plan(4)
+  t.doesNotThrow(function () {
+    fc.assert(
+      fc.property(
+        fc.record({type: fc.string({minLength: 1})}),
+        function (node) {
+          return is(node)
+        }
+      )
+    )
+  }, 'should check if given node (#1)')
+
+  t.doesNotThrow(function () {
+    fc.assert(
+      fc.property(
+        fc.dictionary(fc.string(), fc.string()).filter(function (node) {
```

Comment:
this seems to be doing a ton of work for nothing: no `node` has a `.type`?

Reply:
right, this is a generalized form of
https://github.com/syntax-tree/unist-util-is/blob/18bbf9081ade2cc26eb7f3958c5cbdfbd3d9fc91/test.js#L76
It can check every possible combination of node-like objects, except ones that include `type`